t/24-worker-jobs.t

```diff
@@ -104,7 +104,7 @@ sub wait_until_uploading_logs_and_assets_concluded {
     has sent_messages        => sub { [] };
     has websocket_connection => sub { OpenQA::Test::FakeWebSocketTransaction->new };
     has ua                   => sub { Mojo::UserAgent->new };
-    has url                  => sub { Mojo::URL->new };
+    has url                  => sub { Mojo::URL->new };                                # uncoverable statement count:2
```

Comment:
I've just tried removing this line completely and the test still passes. So I suggest to remove the line completely.

Reply:
even better :)

Reply:
done